pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -128,6 +128,29 @@ def test_sync(self):
         self.assertEqual(latest_version_href, repo.latest_version_href)
         self.assertDictEqual(get_content_summary(repo.to_dict()), RPM_FIXTURE_SUMMARY)
 
+    def test_sync_local(self):
+        """Test syncing from the local filesystem."""
+        import shutil
+
+        if not shutil.which("wget"):
+            unittest.skip("Cannot run file:// sync tests without 'wget' available.")
+
+        cli.Client(self.cfg).run(
+            (
+                "wget",
+                "--recursive",
+                "--no-parent",
+                "--no-host-directories",
+                "--directory-prefix",
+                "/tmp",
```

Comment:
Do we know /tmp is available under the CI?

Reply:
I think it is according to Brian.  For the import export tests IIRC

Reply:
Ah - yes, fair enough, those tests even come with a warning note : https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/tests/functional/api/test_pulpexport.py#L4